test/utils_spec.js

```diff
@@ -33,6 +33,7 @@ describe("utils", function () {
   afterEach(function () {
     cloudinary.config(defaults({
       secure: null,
+      analytics: false,
```

Comment:
I see too many `analytics: false`, consider disabling it globally for all tests

Reply:
I wanted to avoid "if in test mode..." inside our source code. 
Since the configuration is created for every test, I think it's useful to keep it explicit (less magic)

Reply:
this less magic approach is not error proof and will require to have maintenance for each new feature we add(Test class), we also need to consider to disable analytics. and when we have more features like analytics, it can easily go out of control.

For this reason, I suggest one place, where we can disable it for all tests. That can be made by forcing all test to use one base class, or moving this to a helper method, like ResetTestsConfig or something and use it in all methods.

Reply:
Agree with Costa that we need some global way of disabling this feature for tests, and enable it only when relevant.